mmv1/third_party/terraform/services/compute/resource_compute_region_security_policy_rule_test.go.tmpl

```diff
@@ -1,5 +1,4 @@
 package compute_test
-{{- if ne $.TargetVersionName "ga" }}
```

Comment:
`TestAccComputeRegionSecurityPolicyRule_regionSecurityPolicyRuleNetworkMatchUpdate` references `google_compute_network_edge_security_service` which is still beta only, so that test will need to be wrapped with this template conditional.

Reply:
Done